modules/nw-understanding-networking-securing-network-traffic.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * networking/understanding-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-understanding-networking-securing-network-traffic_{context}"]
+= Securing network traffic
+
+{product-title} provides tools to secure your network by creating rules that control which components are allowed to communicate. This is primarily managed through two types of policy resources: network policies and administrative network policies.
+
+== Network policies
```

Comment:
All titles and section headings must have an anchor ID.

OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings